test/service/test_pagination.py

```diff
@@ -129,26 +130,29 @@ def test_search_after_last_page(self):
         Tests that the last page returned in search_after pagination mode is correct.
         :return:
         """
-        content = requests.get("{}?sort=entryId&order=asc".format(self.get_base_url())).content
+        content = requests.get(self.get_base_url() + '?sort=entryId&order=asc').content
         json_response = json.loads(content)
         self.assert_page1_correct(json_response)
         # Store the search_after for the last result of the first page.
         search_after_lrfp = json_response['pagination']['search_after']
         search_after_lrfp_uid = json_response['pagination']['search_after_uid']
-        content = requests.get("{}?sort=entryId&order=asc&search_after={}&search_after_uid={}"
-                               .format(self.get_base_url(), search_after_lrfp, search_after_lrfp_uid)).content
+        content = requests.get(f'{self.get_base_url()}'
+                               f'?sort=entryId'
+                               f'&order=asc'
+                               f'&search_after={search_after_lrfp}'
+                               f'&search_after_uid={search_after_lrfp_uid}').content
         json_response_second = json.loads(content)
         self.assert_page2_correct(json_response, json_response_second, "asc")
 
         search_after = json_response_second['pagination']['search_before']
         search_after_uid = json_response_second['pagination']['search_before_uid']
 
-        content = requests.get(f"{self.get_base_url()}"
-                               f"?sort=entryId"
-                               f"&order=desc"
-                               f"&search_after={search_after}"
-                               f"&search_after_uid={search_after_uid}"
-                               f"&order=desc").content
+        content = requests.get(f'{self.get_base_url()}'
```

Comment:
```suggestion
        content = requests.get(self.get_base_url() +
```

Reply:
I like Daniel's suggestion. The first line looks different to the subsequent ones anyway.